script.js

```diff
@@ -0,0 +1,19 @@
+const chatBotObject = {input:"How are you",output:"Great!"}
+function reply() {
+    let question = document.getElementById("input").value
+    if (chatBotObject.input === question) {
+        document.getElementById("output").value = chatBotObject.output
+    } else {
+        document.getElementById("output").value = "I don't understand that command. Please enter another."
+    }
+    
+}
+
+document.getElementById("submit").addEventListener("click", function() {reply()})
```

Comment:
For your callback function, it's not necessary to create an anonymous function (a function with no name, i.e. `function() { ... }`) and call `reply()` inside of it. It works, but you can just pass in the name of your `reply` function directly. Like this:

```javascript
 document.querySelector("#submit").addEventListener("click", reply);
```

Or your callback function can be an anonymous function like below. If you do this, you don't need to write a separate `reply` function because you have included it inline:
```javascript
 document.querySelector("#submit").addEventListener("click", function() {
   const question = document.querySelector("#input");
   if(question.value === "hello"){
       document.querySelector("#output").textContent = "hi";
   } else{
       document.querySelector("#output").textContent = "I don't understand that command. Please enter another.";
   }
});
```